src/main/java/scala/compat/java8/ScalaStreamSupport.java

```diff
@@ -183,7 +183,7 @@ public static DoubleStream doubleStreamValues(Map<?, Double> coll) {
      * @return     A DoubleStream view of the collection which, by default, executes sequentially.
      */
     public static DoubleStream doubleStreamAccumulated(IterableOnce<Double> coll) {
-        return StreamConverters.asJavaSeqDoubleStream(scala.jdk.DoubleAccumulator$.MODULE$.fromSpecific((IterableOnce<Object>)(Object)coll));
+        return StreamConverters.asJavaSeqDoubleStream(coll);
```

Comment:
we should keep the double accumulator here. but in any case, i already had a draft pr with these changes: https://github.com/scala/scala-java8-compat/pull/143. so closing this one. sorry..!